File: cpp/49a_1372A_OmkarAndCompetition.cpp
#include<bits/stdc++.h>

std::vector<int> solve(int n){
    std::vector<int> arr;
    while(arr.size() < n){
        arr.push_back(1);
    }
    return arr;
}

int main(){
    int t, n;
    std::vector<std::vector<int>> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>n;
        ans.push_back(solve(n));
    }
    for (auto i : ans){
        for(auto j : i){
            std::cout<<j<<" ";
        }
        std::cout<<"\n";
    }
    return 0;
}

File: python/49a_1372A_OmkarAndCompetition.py
def solve(n):
    arr = []
    for i in range(n):
        arr.append(1)
    return arr

def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        ans.append(solve(n))
    for i in ans:
        for j in i:
            print(j, end = " ")
        print()

if __name__ == "__main__":
    main()